serve/services/contribution.js

```diff
@@ -0,0 +1,91 @@
+import { zKey } from 'snarkjs';
+import AWS from 'aws-sdk';
+import logger from '../utils/logger.js'
+import {
+  validateContribution,
+} from '../services/validations.js';
+import { upload } from './upload.js';
+import { isTokenValid, updateCircuitInToken } from '../utils/tokenContributionUtil.js';
+import { CIRCUITS  } from '../utils/constants.js';
+
+export async function applyBeaconContribution(req, token, circuit, data) {
```

Comment:
Are all these abstractions needed? `applyBeaconContribution`, `applyUserContribution` etc?
They dont seem to provide much more value than an name alias.

Reply:
I know, they don't do much, but this avoids someone from calling the method with the wrong parameters. This helps to make things a bit clear too, and the main method is not exported.